frost-core/src/keys.rs

```diff
@@ -1,5 +1,7 @@
 //! FROST keys, keygen, key shares
 #![allow(clippy::type_complexity)]
+// Remove after https://github.com/rust-lang/rust/issues/147648 is fixed
+#![allow(unused_assignments)]
```

Comment:
```suggestion
#![expect(unused_assignments)]
```
Using `expect` will automatically trigger a warning when the compiler is fixed

Reply:
TIL, thanks! However, that will emit warnings for whoever is using the crate in pre-1.92 Rust :frowning_face: